src/style.css

```diff
@@ -0,0 +1,27 @@
+* {
+  margin: 0;
```

Comment:
you can put only box-sizing to  this`*{}` block, because styles from this block will be applied to every elements